### PR TITLE
document bytes signing

### PR DESCRIPTION
**Pull request by ericHgorski**
@evanorti I created a new wallet-provider section under SDKs. let me know if theres any additional formatting required there.

Will also be a good place for the getting started with wallet provider guide Id worked on.

### CODE REVIEW

File: docs/develop/sdks/wallet-provider/sign-bytes.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Signing Bytes
+
+This document explains how to sign arbitrary bytes with Wallet Provider in a react-based web application. This action is useful for verifying account ownership without having to post a transaction to the chain. It's commonly used as a form of simple user authentication.
```
Comment (santerra505) [inline docs/develop/sdks/wallet-provider/sign-bytes.md]:
```suggestion
You can sign arbitrary bytes with [Wallet Provider](https://www.npmjs.com/package/@terra-money/wallet-provider) in a React-based web application. This action is useful for verifying account ownership without having to post a transaction to the chain, and is commonly used as a form of simple user authentication.
```

Suggested edits:
1) Link to Wallet Provider page
2) Capitalize "react" 
3) Phrasing

File: docs/develop/sdks/wallet-provider/sign-bytes.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+*You can find this code used in context [here](https://github.com/terra-money/wallet-provider/blob/main/templates/create-react-app/src/components/SignBytesSample.tsx). Not using react? Use [wallet-controller](https://www.npmjs.com/package/@terra-money/wallet-controller)*.
```
Comment (santerra505) [inline docs/develop/sdks/wallet-provider/sign-bytes.md]:
```suggestion
:::{tip}
Not using React? Use the [wallet-controller](https://www.npmjs.com/package/@terra-money/wallet-controller) instead.
:::
```

Change requested: use tip box

I would also suggest moving the first sentence about the SignBytesSample code in context to another part of this file, as it seems like a separate idea from the first sentence.

File: docs/develop/sdks/wallet-provider/sign-bytes.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+Wallet Provider comes with a `useConnectedWallet` hook which let's you trigger actions from a Terra wallet that's connected to the web page. The `connectedWallet` object is includes a `.signBytes()` method which prompts the user to sign the data and then returns an object of type `SignBytesResult`. This object contains the address of the signer and the corresponding signed data.
```
Comment (santerra505) [inline docs/develop/sdks/wallet-provider/sign-bytes.md]:
```suggestion
The Wallet Provider comes with a `useConnectedWallet` hook, which lets you trigger actions from a Terra wallet that's connected to the web page. The `connectedWallet` object includes a `.signBytes()` method, which prompts the user to sign the data and then returns an object of type `SignBytesResult`. The returned `SignBytesResult` object contains the address of the signer and the signed data.
```

Minor nitpicks and phrasing. Note that the edit fixes two typos (bolded) _...the connectedWallet object is includes a .signBytes() method..._ and _which **let's** you trigger actions from a Terra wallet_

File: docs/develop/sdks/wallet-provider/sign-bytes.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+*You can find this code used in context [here](https://github.com/terra-money/wallet-provider/blob/main/templates/create-react-app/src/components/SignBytesSample.tsx). Not using react? Use [wallet-controller](https://www.npmjs.com/package/@terra-money/wallet-controller)*.
+
+Wallet Provider comes with a `useConnectedWallet` hook which let's you trigger actions from a Terra wallet that's connected to the web page. The `connectedWallet` object is includes a `.signBytes()` method which prompts the user to sign the data and then returns an object of type `SignBytesResult`. This object contains the address of the signer and the corresponding signed data. 
+
+The `verifyBytes` function then compares the original `TEST_BYTES` against the signature and public key pairing provided by the `SignBytesResult`. If it returns `true`, then the account is indeed under the owernship of the connected wallet and vice versa. In this way, the owner of the associated wallet is verified without having to produce an on-chain action or pay gas fees.
```
Comment (santerra505) [inline docs/develop/sdks/wallet-provider/sign-bytes.md]:
```suggestion
The `verifyBytes` function then compares the original `TEST_BYTES` against the signature and public key pairing returned by the `SignBytesResult`. If `verifyBytes` returns `true`, then the account is owned by the connected wallet. Likewise, if `verifyBytes` returns `false`, then the account is not owned by the connected wallet. In this way, the owner of the associated wallet is verified without having to produce an on-chain action or pay gas fees.
```

A few suggestions:
1) Explicitly state what happens when `false` is returned
2) Phrasing

File: docs/develop/sdks/wallet-provider/sign-bytes.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+*You can see how the `verifyBytes` function works under the hood [here](https://github.com/terra-money/wallet-provider/blob/4e601c2dece7bec92c9ce95991d2314220a2c954/packages/src/%40terra-money/wallet-controller/verifyBytes.ts#L1).*
+
+Finally, Wallet Provider also supplies useful error types that can be used in conjunction with a `catch` to direct or notify the user of whether the signing was sucessful.
```
Comment (santerra505) [inline docs/develop/sdks/wallet-provider/sign-bytes.md]:
```suggestion
Wallet Provider also supplies useful error types that can be used with a `catch` statement notify the user whether or not the signing was successful.
```

File: docs/develop/sdks/wallet-provider/sign-bytes.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+Wallet Provider comes with a `useConnectedWallet` hook which let's you trigger actions from a Terra wallet that's connected to the web page. The `connectedWallet` object is includes a `.signBytes()` method which prompts the user to sign the data and then returns an object of type `SignBytesResult`. This object contains the address of the signer and the corresponding signed data. 
+
+The `verifyBytes` function then compares the original `TEST_BYTES` against the signature and public key pairing provided by the `SignBytesResult`. If it returns `true`, then the account is indeed under the owernship of the connected wallet and vice versa. In this way, the owner of the associated wallet is verified without having to produce an on-chain action or pay gas fees.
+
+*You can see how the `verifyBytes` function works under the hood [here](https://github.com/terra-money/wallet-provider/blob/4e601c2dece7bec92c9ce95991d2314220a2c954/packages/src/%40terra-money/wallet-controller/verifyBytes.ts#L1).*
```
Comment (santerra505) [inline docs/develop/sdks/wallet-provider/sign-bytes.md]:
```suggestion
:::{tip}
You can see how the `verifyBytes` function works [here](https://github.com/terra-money/wallet-provider/blob/4e601c2dece7bec92c9ce95991d2314220a2c954/packages/src/%40terra-money/wallet-controller/verifyBytes.ts#L1).*
:::
```

Change requested: 
use `:::{tip}`

**Review by santerra505 — CHANGES_REQUESTED** [review]

**Comment by evanorti** [comment]
LGTM!

**Pull request merged**